ui/components/MesheryPatterns.js

```diff
@@ -634,30 +634,30 @@ function MesheryPatterns({
       },
     );
     catalogVisibilityRef.current = catalogVisibility;
-    const fetchCatalogPatterns = fetchCatalogPattern({
-      selector: {
-        search: '',
-        order: '',
-        page: 0,
-        pagesize: 0,
-      },
-    }).subscribe({
-      next: (result) => {
-        catalogContentRef.current = result?.catalogPatterns;
-        initPatternsSubscription();
-      },
-      error: (err) => console.log('There was an error fetching Catalog Filter: ', err),
-    });
-
-    return () => {
-      fetchCatalogPatterns.unsubscribe();
-      disposeConfSubscriptionRef.current?.dispose();
-    };
+    // const fetchCatalogPatterns = fetchCatalogPattern({
```

Comment:
@sudhanshutech will you document why this section remains commented and not deleted?

It's fine that it is not deleted, but document why is it commented out and why it's still around, please.